app/models/version.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+class Version < ActiveRecord::Base
+  belongs_to :user, inverse_of: :versions
```

Comment:
Related to @patrickvinograd's User model comment...you might consider naming it `:created_by` instead of `:user` to reveal the intent better (here and in the User model.)